### PR TITLE
Adding scannotations to orbit

### PR DESCRIPTION
**Pull request by ruwany**
Adding scannotations to read annotations

### CODE REVIEW

File: scannotation/1.0.3.wso2v1/pom.xml [hunk 1]
```diff
@@ -0,0 +1,90 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+ ~ Copyright (c) WSO2 Inc. (http://wso2.com) All Rights Reserved.
```
Comment (sameera-jayasoma) [inline scannotation/1.0.3.wso2v1/pom.xml]: Year is missing

File: scannotation/1.0.3.wso2v1/pom.xml [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+ ~ See the License for the specific language governing permissions and
+ ~ limitations under the License.
+-->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>org.wso2.scannotation</groupId>
```
Comment (sameera-jayasoma) [inline scannotation/1.0.3.wso2v1/pom.xml]: group ID is not correct. 

it should be org.wso2.orbit.org.scannotation

File: scannotation/1.0.3.wso2v1/pom.xml [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+    <version>1.0.3.wso2v1</version>
+    <packaging>bundle</packaging>
+    <name>Scannotation orbit bundle</name>
+    <description>scannotation</description>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.scannotation</groupId>
+            <artifactId>scannotation</artifactId>
+            <version>${org.scannotation.version}</version>
```
Comment (sameera-jayasoma) [inline scannotation/1.0.3.wso2v1/pom.xml]: We need to mark these dependencies as optional

File: scannotation/1.0.3.wso2v1/pom.xml [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+                        <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
+                        <Bundle-Name>${project.artifactId}</Bundle-Name>
+                        <Export-Package>
+                            org.scannotation.*;version="1.0.3",
+                            !javassist.*
+                        </Export-Package>
+                        <Import-Package>
+                            javassist.*
+                        </Import-Package>
+                        <DynamicImport-Package>*</DynamicImport-Package>
```
Comment (sameera-jayasoma) [inline scannotation/1.0.3.wso2v1/pom.xml]: Can you elaborate more on the need to specify DynamicImport-Package header?

File: scannotation/1.0.3.wso2v1/pom.xml [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+                        <Include-Resource>
+                        </Include-Resource>
+                        <Embed-Dependency>javassist;inline=false</Embed-Dependency>
```
Comment (sameera-jayasoma) [inline scannotation/1.0.3.wso2v1/pom.xml]: If you are embedding javassist librariy, why do we need to import javassist.* package

**Comment by ruwany** [comment]
Did the requested changes. Will send a fresh pull request. Thanks.

**Pull request closed without merging**